cscan.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 pushd "$(dirname ${BASH_SOURCE[0]})" > /dev/null
 if [ ! -d ./tlslite ]; then
-    git clone --depth=1 https://github.com/tomato42/tlslite-ng.git .tlslite-ng
+    git clone --depth=1 https://github.com/tomato42/tlslite-ng.git .tlslite-ng &>/dev/null
```

Comment:
Those network communication can be seen by using the `-D` switch no ?

The first I've used your tool, I was a bit surprised to see this `git` process starting without any information. then, on every use, I was spotted by the an error where I got no idea where that came from. After some reading of your source code, I've found that the issues were in `cscan.sh`.

I'll modify my PR to include some output when these `git` commands are started the first time